tensorflow/core/distributed_runtime/rpc/grpc_worker_service.cc

```diff
@@ -337,32 +337,25 @@ void GrpcWorker::RecvTensorAsync(CallOptions* opts,
             if (src_dev->tensorflow_gpu_device_info() && (!on_host)) {
 #if GOOGLE_CUDA
               const DeviceContext* send_dev_context = send_args.device_context;
-              RecvTensorResponse* tmp = new RecvTensorResponse;
-              tmp->set_is_dead(is_dead);
+              AllocatorAttributes alloc_attrs;
```

Comment:
Ok, thanks! To run the benchmark:
1. Run `./configure` and enable CUDA.
2. `bazel build -c opt tensorflow/core/distributed_runtime:rpcbench_test`
3. `bazel-bin/tensorflow/core/distributed_runtime/rpcbench_test --benchmarks=all > /tmp/benchmark_output`

Read and compare the before and after files for benchmark output.